### DISCUSSION TITLE
Official Homebrew tap

### DISCUSSION THREAD

**Discussion by unitof** [ruffle-rs/ruffle#5902] [Ideas] [answered]
Hi!

I'm a bigtime Homebrew user and made a `ruffle` tap: https://github.com/unitof/homebrew-ruffle/

Curious if the maintainers around these parts would be interested in making it [official](https://github.com/ruffle-rs/homebrew-ruffle/). I'd be happy to maintain.

Once there's more normalized versioning/bundling, that can easily be added. And when it's considered "stable" it could be added to homebrew/core! I'd love to do that honor when the time comes!

**Comment by Herschel** [comment]
Hi, sorry for the delay in responding here, and thanks for making a Homebrew tap!

Sure, I'd love to make this official if you'd be willing to help maintain it. I'm not too familiar with Homebrew, but based on your repo it seems like it'd be fairly straightforward -- would that repo itself move to live as a repo in the Ruffle org?

Thank you!

**Reply by unitof** [reply]
Hey, no worries! Yes, I'd still really love to.

Until it is versioned enough to move to homebrew/homebrew-core, yep that's what I'm proposing:

1. Fork or transfer (your choice) unitof/homebrew-ruffle to ruffle-rs/homebrew-ruffle (exact name is important for Homebrew).

2. Grant me write access to at least that repo in ruffle-rs org and I'll maintain it.

What's the current status/ETA on versioned Ruffle releases (of any semantic or stability), instead of just that latest Git HEAD? Just asking since that slightly changes how I'd maintain the tap.

<small>Long live Homestar Runner!</small>

**Reply by unitof** [reply]
I lean toward transferring so I don't have an outdated one, and anyone using mine would be auto-redirected to the new one ([GitHub feature](https://github.community/t/how-long-does-github-forward-renamed-moved-repos/582)).

**Reply by unitof** [reply]
Hi @Herschel, just nudging. I just tried to initiate a transfer request but turns out I cannot without rights to create public repos in `ruffle-rs`.

**Reply by Herschel** [reply]
Hi @unitof, tossed you an invite + an e-mail. Thank you!

**Comment by unitof** [comment, marked as answer]
https://github.com/ruffle-rs/homebrew-ruffle is now official!
